packages/alias/test/test.js

```diff
@@ -26,204 +67,93 @@ test('defaults', (t) => {
   t.is(typeof result.resolveId, 'function');
 });
 
-test('Simple aliasing (array)', (t) => {
-  const result = alias({
-    entries: [
-      { find: 'foo', replacement: 'bar' },
-      { find: 'pony', replacement: 'paradise' },
-      { find: './local', replacement: 'global' }
-    ]
-  });
-
-  const resolved = result.resolveId('foo', '/src/importer.js');
-  const resolved2 = result.resolveId('pony', '/src/importer.js');
-  const resolved3 = result.resolveId('./local', '/src/importer.js');
-
-  t.is(resolved, 'bar');
-  t.is(resolved2, 'paradise');
-  t.is(resolved3, 'global');
-});
-
-test('Simple aliasing (object)', (t) => {
-  const result = alias({
-    entries: {
-      foo: 'bar',
-      pony: 'paradise',
-      './local': 'global'
-    }
-  });
-
-  const resolved = result.resolveId('foo', '/src/importer.js');
-  const resolved2 = result.resolveId('pony', '/src/importer.js');
-  const resolved3 = result.resolveId('./local', '/src/importer.js');
-
-  t.is(resolved, 'bar');
-  t.is(resolved2, 'paradise');
-  t.is(resolved3, 'global');
-});
-
-test('RegExp aliasing', (t) => {
-  const result = alias({
-    entries: [
-      { find: /f(o+)bar/, replacement: 'f$1bar2019' },
-      { find: new RegExp('.*pony.*'), replacement: 'i/am/a/barbie/girl' },
-      { find: /^test\/$/, replacement: 'this/is/strict' }
-    ]
-  });
-
-  const resolved = result.resolveId('fooooooooobar', '/src/importer.js');
-  const resolved2 = result.resolveId('im/a/little/pony/yes', '/src/importer.js');
-  const resolved3 = result.resolveId('./test', '/src/importer.js');
-  const resolved4 = result.resolveId('test', '/src/importer.js');
-  const resolved5 = result.resolveId('test/', '/src/importer.js');
-
-  t.is(resolved, 'fooooooooobar2019');
-  t.is(resolved2, 'i/am/a/barbie/girl');
-  t.is(resolved3, null);
-  t.is(resolved4, null);
-  t.is(resolved5, 'this/is/strict');
-});
-
-test('Will not confuse modules with similar names', (t) => {
-  const result = alias({
-    entries: [
-      { find: 'foo', replacement: 'bar' },
-      { find: './foo', replacement: 'bar' }
-    ]
-  });
-
-  const resolved = result.resolveId('foo2', '/src/importer.js');
-  const resolved2 = result.resolveId('./fooze/bar', '/src/importer.js');
-  const resolved3 = result.resolveId('./someFile.foo', '/src/importer.js');
-
-  t.is(resolved, null);
-  t.is(resolved2, null);
-  t.is(resolved3, null);
-});
-
-test('Local aliasing', (t) => {
-  const result = alias({
-    entries: [
-      { find: 'foo', replacement: './bar' },
-      { find: 'pony', replacement: './par/a/di/se' }
+test('Simple aliasing (array)', (t) =>
+  resolveAliasWithRollup(
+    {
+      entries: [
+        { find: 'foo', replacement: 'bar' },
+        { find: 'pony', replacement: 'paradise' },
+        { find: './local', replacement: 'global' }
+      ]
+    },
+    [['foo', '/src/importer.js'], ['pony', '/src/importer.js'], ['./local', '/src/importer.js']]
```

Comment:
Personally would prefer for a single test to be an object (`{ requester, importSource }` or similar) rather than a tuple, but if you prefer tuples I won't insist on this one.

Reply:
Agreed, makes the test probably a bit more readable. Maybe stick with the terminology used in Rollup's docs: https://rollupjs.org/guide/en/#resolveid which would be `{source, importer}`